pyproject.toml

```diff
@@ -84,5 +84,18 @@ doc = [
 [project.scripts]
 ihatemoney = "ihatemoney.manage:cli"
 
-[tool.setuptools]
-packages = ["ihatemoney"]
+[tool.hatch.build.hooks.custom]
+dependencies = [
+    "Babel>=2.10.1"
```

Comment:
This is the oldest babel python I could find that would support all the python versions supported by ihatemoney.
python 3.10 support has been brought with babel 2.10, using babel<2.10 would have broke minversions tests for python 3.10+